lib/lunchbot/lunchbot_data.ex

```diff
@@ -587,7 +587,7 @@ defmodule Lunchbot.LunchbotData do
   end
 
   def format_orders(order_struct) do
```

Comment:
You can overload `format_orders` to handle these cases instead of embedding the whole method in an if statement:

```elixir
def format_orders(order_struct = [%Order{} | _])
def format_orders(_) do: nil
```

Reply:
Cool, I didn't know I could also do that. Thanks for the recommendation!